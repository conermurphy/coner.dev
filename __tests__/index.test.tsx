import { render } from '@testing-library/react';
import Home from '../pages/index';

jest.mock('next/router', () => {
  return {
    useRouter: () => {
      return {
        query: { myProp: 'myValue' },
        pathname: '/',
      };
    },
  };
});

jest.mock('next/link', () => {
  return ({ children }: { children: JSX.Element }) => {
    return children;
  };
});

it('homepage matches snapshot', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
