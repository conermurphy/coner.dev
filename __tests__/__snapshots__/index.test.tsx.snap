// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage matches snapshot 1`] = `
<div>
  <div>
    <h1>
      Coner Murphy
    </h1>
  </div>
</div>
`;
